src/sms-verification/templates/user_management/sms-verification/verify_form.html

```diff
@@ -0,0 +1,25 @@
+<div class="alert alert-success" ng-show="verified">
+    <p translate>Verification successful.</p>
+</div>
+<div class="alert alert-success" ng-show="resent">
+    <p translate>Activation code resent.</p>
+</div>
+
+<div class="form-group" ng-class="{'has-error': errors.phone_number}">
+    <label for="phone_number" translate>Phone number</label>
+    <input type="phone_number" ng-model="data.phone_number" class="form-control" id="phone_number">
```

Comment:
Is this `type="phone_number" `trying to use the html5 input types? I though it was `<input type="tel">` for a phone number?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-type
http://html5doctor.com/html5-forms-input-types/#input-tel